bash_completions.go

```diff
@@ -100,6 +101,12 @@ __handle_filename_extension_flag()
     _filedir "@(${ext})"
 }
 
+__handle_subdirs_in_dir_flag()
+{
+    local dir="$1"
+    cd "${dir}" >/dev/null 2>&1 && _filedir -d && cd - >/dev/null
```

Comment:
I prefer `pushd` and `popd` instead of `cd` `cd -` as push/pop don't get confused if cwd changes are made by the intervening operations. In this case it doesn't look like _filedir() bash completions code uses cd/push either, so it would be fine. But since half of the improvements to open source are small iterations of the work which has come before, I'd rather your code be harder for someone else to get wrong in the future.